providers/aws/resources/aws.lr

```diff
@@ -2080,7 +2080,7 @@ private aws.rds.backupsetting {
 }
 
 // Amazon RDS database cluster
-private aws.rds.dbcluster @defaults("id region") {
+private aws.rds.dbcluster @defaults("id region engine engineVersion") {
```

Comment:
since we can call the resource now, we should remove `private`

Reply:
dbinstance is still private so I kept this one private as well